src/definitions_cxx.hpp

```diff
@@ -1084,6 +1084,7 @@ constexpr int32_t MIDI_CHANNEL_NONE = 255;
 constexpr int32_t MIDI_CC_NONE = 255;
 
 constexpr int32_t IS_A_CC = NUM_CHANNELS;
+constexpr int32_t IS_A_PC = IS_A_CC + NUM_CHANNELS; // CC128 is max.
```

Comment:
In retrospect from the CC hack this doesn't scale well - it'd be better to make it 2 << 24 and then just & it with the int32 (but really 5 bit) channel

That way checking if a saved midi command is a PC is note & 0xFF000000 == IS_A_PC and we don't have to worry about what happens if we need more types in the future

Reply:
I believe that we shouldn't break consistency with the CC hack, because then we'll have two different hacks. We should fix the CC and this at the same time using the approach you are advocating for here. That way we eliminate all the hacks and don't introduce inconsistency through this PR.

Reply:
yep reasonable